src/daft-proto/proto/v1/daft.proto

```diff
@@ -0,0 +1,483 @@
+syntax = "proto3";
+
+package daft.v1;
+
+// -------------------------------------------------------------
+//
+//                          SCHEMA
+//
+// -------------------------------------------------------------
+
+message Schema {
+  repeated Field fields = 1;
+}
+
+message DataType {
+  oneof data_type_variant {
+    bool null = 1;
```

Comment:
instead of bool i would suggest making an `EmptyType` so the value won't be misinterpreted